docs/usage_guide/advanced_usage/modin_logging.rst

```diff
@@ -0,0 +1,41 @@
+Logging and Tracing with Modin
+==============================
+
+Modin logging and tracing allows users to gain observability into their queries by logging internal Modin API calls, partition metadata, 
+and memory profiling. When Modin logging is enabled (default disabled), log files are written to a local `.modin` directory at the same
+directory level as the notebook/script used to run Modin. Each log file in the `.modin` directory is named after the job uuid. 
+
+Usage example
+-------------
+
+In the example below, we enable logging for internal Modin API calls. 
+
+.. code-block:: python
+
+  import modin.pandas as pd
+  from modin.config import LogMode
+  LogMode.enable_api_only()
+
+  # Your code goes here
+
+In this example, we enable logging for internal Modin API calls, partition metadata, and memory profiling.
```

Comment:
This gets a little repetitive since its essentially the same thing.
```suggestion
In the next example, we add logging not only for internal API calls, but also for partition metadata and memory profiling.```